docs/UserGuide.adoc

```diff
@@ -12,19 +12,24 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level4
+:repoURL: https://github.com/CS2103-AY1819S1-T09-2/Main.git
 
-By: `T09-02`      Since: `Sep 2018`      Licence: `MIT`
+By: `Team T09-2`      Since: `Aug 2018`      Licence: `MIT`
 
 == Introduction
+Welcome to the *Anakin* User Guide!
 
-*Anakin*  is for those who *want to make heavy use of flashcards without the hassle of managing physical copies*. More importantly, Anakin is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+*Anakin*  was built for those who *want to make heavy use of flashcards without the hassle of managing physical copies*. More importantly, Anakin is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI).
```

Comment:
I think we can add in a few lines talking about `Purpose of User Guide` (CS2101 requirements). 

Example: 
>  This user guide provides in-depth documentation on the Anakin installation process, features...